core/gallery/gallery_test.go

```diff
@@ -23,3 +25,101 @@ func TestIsInWhitelist(t *testing.T) {
 	assert.True(t, IsInWhitelist("anytype://anytype.io/localstorage/knowledge_base.zip"))
 	assert.True(t, IsInWhitelist("anytype://gallery.any.coop/"))
 }
+
+func TestDownloadManifest(t *testing.T) {
+	t.Run("download knowledge base manifest", func(t *testing.T) {
+		// given
+		url := "https://raw.githubusercontent.com/anyproto/gallery/main/experiences/knowledge_base/manifest.json"
+
+		// when
+		info, err := DownloadManifest(url)
```

Comment:
does it send real request? Test must be runnable locally without internet connection

Reply:
Added local http server to test manifest downloading and schema validation